oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -353,6 +361,21 @@ static int spml_ucx_init(void)
                        -1, EV_PERSIST, mca_spml_ucx_async_cb, NULL);
     }
 
+    if (mca_spml_ucx.symmetric_rkey) {
+        if (mca_spml_ucx.symmetric_rkey_size <= 0) {
+            SPML_UCX_ERROR("failed to create symmetric rkey store with size %d",
+                           mca_spml_ucx.symmetric_rkey_size);
+            return OSHMEM_ERROR;
+        }
+
+        ret = mca_spml_ucx_rkey_store_create(&mca_spml_ucx.rkey_store,
+                                             mca_spml_ucx_ctx_default.ucp_worker[0],
+                                             mca_spml_ucx.symmetric_rkey_size);
```

Comment:
1. as we discussed, probably need rkey store on each shmem context
2. maybe no need to pass the store size as a parameter? when needed, we can access mca_spml_ucx.symmetric_rkey_size variable (that IMO should be renamed to mca_spml_ucx.symmetric_rkey_max_count or similar)

Reply:
fixed